app/views/refills/_nav.html.erb

```diff
@@ -32,6 +31,18 @@
           <%end%>
         </form>
       </div>
+      <%elsif current_page?(controller: 'events', action: 'new')%>
```

Comment:
can you maybe move this component outside of the shared nav partial and only render it on pages you need it?